mlir_graphblas/explorer.py

```diff
@@ -0,0 +1,151 @@
+import string
+import random
+from pygments import highlight
+from pygments.formatters import get_formatter_by_name
+from pygments import token, lexer
+from functools import partial
+
+
+class MlirLexer(lexer.RegexLexer):
```

Comment:
We should cite the Chroma file that these regex patterns came from in a comment.

Reply:
Good catch. Added a comment.